pkg/login/auth.go

```diff
@@ -18,9 +20,11 @@ var (
 	ErrGettingUserQuota      = errors.New("Error getting user quota")
 )
 
-func Init() {
-	bus.AddHandler("auth", AuthenticateUser)
-	loadLdapConfig()
+func init() {
+	lifecycle.AddListener(lifecycle.ApplicationStarting, func() {
```

Comment:
Why do we need this? The application lifecycle management becomes more and more complex.